src/bots/reddit/utils.py

```diff
@@ -12,7 +12,7 @@ def parse_user(user: Redditors):
   i['link_karma'] = user.link_karma
   i['username'] = user.name
   i['created_utc'] = user.created_utc
-
+  i['created_utc_human'] = time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(user.created_utc))
```

Comment:
I would prefer it we store all times utc rather than making conversions while stored. it can make it harder to troubleshoot issues, and it's easier to do the conversion as needed. such as when showing the value to the user in logs or a UI